packages/twenty-front/src/modules/ui/utilities/state/component-state/utils/createComponentFamilySelectorV2.ts

```diff
@@ -9,7 +11,26 @@ import { SelectorGetter } from '@/ui/utilities/state/types/SelectorGetter';
 import { SelectorSetter } from '@/ui/utilities/state/types/SelectorSetter';
 import { isDefined } from 'twenty-ui';
 
-export const createComponentFamilySelectorV2 = <
+export function createComponentFamilySelectorV2<
```

Comment:
@lucasbordeau seems non standard (lint rules)

Reply:
This is function overloading in TypeScript we cannot do it with const